fibonacci_number/FibonacciNumber.java

```diff
@@ -0,0 +1,27 @@
+public class FibonacciNumber {
+
+    public static void fibonacci(int n) {
```

Comment:
Please make the function to return the result and print it in the `main` method only, just to keep it consistent with the rest of the code.